app/src/main/java/com/sun/wsm_android_v2/data/local/UserLocalDataSoure.kt

```diff
@@ -0,0 +1,19 @@
+package com.sun.wsm_android_v2.data.local
+
+import com.sun.wsm_android_v2.data.UserDataSource
+import com.sun.wsm_android_v2.data.local.sharedprf.SharedPrefsApi
+import com.sun.wsm_android_v2.data.local.sharedprf.SharedPrefsKey
+import javax.inject.Inject
+
+class UserLocalDataSoure @Inject constructor(
```

Comment:
a đã add sẵn trong module nên không cần inject đâu e
```suggestion
class UserLocalDataSoure constructor(
```